scicamera/controls.py

```diff
@@ -1,63 +1,45 @@
-#!/usr/bin/python3
 from __future__ import annotations
 
-import threading
 from typing import Set
 
 from libcamera import ControlType, Rectangle, Size
 
 
-def _framerates_to_durations_(framerates):
+def _framerates_to_durations(framerates):
     if not isinstance(framerates, (tuple, list)):
         framerates = (framerates, framerates)
     return (int(1000000 / framerates[1]), int(1000000 / framerates[0]))
 
 
-def _durations_to_framerates_(durations):
+def _durations_to_framerates(durations):
     if durations[0] == durations[1]:
         return 1000000 / durations[0]
```

Comment:
Oh, man, this is a really bad pattern. Like, really really bad. The return type changes based on the _value_ of the input? 

I know it's not your pattern, but I'd  🔪 stuff like this ASAP.